libminifi/include/core/json/JsonFlowSerializer.h

```diff
@@ -0,0 +1,38 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "yaml-cpp/yaml.h"
+#include "core/flow/FlowSerializer.h"
+
+namespace org::apache::nifi::minifi::core::json {
+
+class JsonFlowSerializer : public core::flow::FlowSerializer {
+ public:
+  explicit JsonFlowSerializer(rapidjson::Document&& document) : flow_definition_json_(std::move(document)) {}
```

Comment:
do we need the rvalue ref here?

Reply:
fixed in 52814c6cc538df8715d22ea213f6f5a355fc0118